test/support/feature_case.ex

```diff
@@ -10,6 +10,8 @@ defmodule KwtoolWeb.FeatureCase do
 
       alias KwtoolWeb.Router.Helpers, as: Routes
 
+      @moduletag :feature_test
```

Comment:
Are we doing something with this tag 👀 ?

Reply:
yeah, just sometimes on my local, i run the test by ` mix test --exclude=feature_test`